bin/picca_co.py

```diff
@@ -181,6 +183,10 @@ def main():
                         help='Number of processors')
 
     args = parser.parse_args()
+    if len(args.drq)==1:
+        args.drq=args.drq[0]
+    if args.drq2 is not None and len(args.drq2)==1:
+        args.drq2=args.drq2[0]
```

Comment:
why are you doing this change? What happens if args.drq has more than one item in the list? Is this managed by io.read_objects ?

Reply:
It's to allow people to use multiple catalogs at the same time, which also allows e.g. running with a bunch of zbest files instead of creating a cat beforehand. The original `read_drq` routine cannot cope with a list of files so I changed it that way, the alternative would be always requiring proper catalogs always. Or to modify the `read_drq` to loop over all elements of a list instead of converting the list when it's only one entry

Reply:
It makes sense to allow the code to run on multiple catalogues. However, as things stand now the code will break if two catalogues are passed, won't it? Or am I missing something? I would also include the necessary changes in `read_drq` so that the code does not break.

Reply:
ok, these changes are removed now.